addons/l10n_za/data/template/account.account-za.csv

```diff
@@ -43,7 +47,11 @@
 "300050","300050","Capital","equity","False"
 "300060","300060","Dividends","equity","False"
 "300070","300070","Drawings","equity","False"
+"360000","360000","Profit and Loss (This Year)","equity_unaffected","False"
```

Comment:
This is the same as 400010?
"Retained Income (Profit and Loss)" 🤔

Reply:
I don't see on the Excel the 400010 account you're speaking about

Reply:
Let's keep the one we already have